game/static/game/css/board.css

```
@@ -1361,15 +1206,33 @@ body {
     flex: 1 1 auto;
-    }
+}
 
-    .clock .time {
-        font-family: 'Consolas', 'Courier New', monospace;
-        font-size: 1.15rem;
-        font-weight: 700;
-        flex-shrink: 0;
-    }
+.clock .time {
+    font-family: 'Consolas', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)